config.js

```diff
@@ -7,6 +7,10 @@
 // entrypoint for the entire applications configuration.
 require('env-rewrite').rewrite();
 
+// Apply all the configuration provided in the .env file if it isn't already
+// in the environment.
+require('dotenv').config();
```

Comment:
Posting this again, cause it dissappeared into the outdated zone:

Doesn't seem to work for me. Where does it look for a `.env` file?

Reply:
In your root talk directory, it will load configuration into the process if it isn't already in the environment.

Reply:
Oh it didn't work for me before, but now it does.